imsv-docs-astro/astro.config.mjs

```diff
@@ -36,16 +36,20 @@ export default defineConfig({
           label: 'Guides',
           items: [
             { label: 'Introduction', link: 'guides' } ,
+            { label: 'Authentication', link: 'guides/authentication' } ,
```

Comment:
just re-arranged so non-toggle items are at the top

Reply:
Please put in a separate PR

Reply:
separate pr means likely no PR

Reply:
That's fine. It's a known issue